.github/workflows/storybook-tests.yml

```diff
@@ -16,9 +16,11 @@ jobs:
       - name: Install Playwright
         run: npx playwright install --with-deps
       - name: Build Storybook
-        run: yarn build-storybook-for-tests
+        run: yarn build-storybook --quiet
+      - name: Run accessibility tests 🤟
+        run: yarn storybook:axe
       - name: Serve Storybook and run tests
         run: |
           npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
-            "npx http-server storybook-static --port 6006 --silent" \
+            "npx http-server docs-build --port 6006 --silent" \
```

Comment:
Updated the Storybook build to always be at `docs-build/`, instead of sometimes there and sometimes at `storybook-static/`.

Reply:
Totally makes sense :D